source/marsGameSound.vwf.yaml

```diff
@@ -20,6 +20,12 @@ properties:
   soundSet:
     - soundName: "musicStandardGameplay"
       soundURL: "assets/sounds/music/musicStandardGameplayLoopFullMix.ogg"
+      useTextToSpeech: true
+      ttsAmplitude: 100
+      ttsVariant: "m1"
+      ttsWordGap: 0
+      ttsSpeed: 175
+      ttsPitch: 50
```

Comment:
Also, merge amplitude into the existing volume control (but maybe let the voice set the base amplitude - or something).